sql/build.gradle

```diff
@@ -10,6 +10,10 @@ configurations {
     benchmarksRuntime.extendsFrom testRuntime, benchmarksCompile
 }
 
+repositories {
+    jcenter()
```

Comment:
consider adding this to the root build.gradle and remove this from all other submodules like app/build.gradle. afaik there'd be no drawback or?